src/components/PostList.vue

```diff
@@ -10,14 +13,12 @@
 import RowDetail from "./RowDetail.vue";
 export default {
   name: "Post-List",
-  props: ["postDetails"],
   components: {
     RowDetail,
   },
-
-  methods: {
-    deletePost(data) {
-      this.$emit("deletePost", data);
+  computed: {
+    postDetails() {
+      return this.$store.getters.getPostDetail;
```

Comment:
We are using `postDetails` in our component and our vuex's getter name is `getPostDetail`.
Consistency is recommended in naming convention.